components/BookDetailsCard/BookDetailsCard.js

```diff
@@ -62,22 +62,26 @@ export default function BookDetailsCard({
   );
 }
 
+const StyledAuthor = styled.p`
+  font-size: 18px;
+`;
+
 const StyledImage = styled(Image)`
   border-radius: 8px;
   position: relative;
   top: -20px;
   box-shadow: 0 6px 20px 5px rgba(0, 0, 0, 0.19);
 `;
 const StyledBookDetail = styled.article`
-  border: 2px solid black;
+  box-shadow: 0 0 20px 5px rgba(0, 0, 0, 0.19);
   border-radius: 8px;
   margin: 2rem auto;
   text-align: center;
+  background-color: var(--color-light-yellow);
 `;
 const StyledSection = styled.ul`
   display: flex;
   justify-content: space-around;
-  font-style: italic;
   list-style: none;
   padding: 0px;
```

Comment:
It is a convention not to use a unit for the value 0.

```suggestion
  padding: 0;
```